### PR TITLE
👌 IMPROVE: Making class names less generic

### PR DESCRIPTION
**Pull request by AakashGfude**
This PR updates the class names used by this theme to be less likely to clash with inherited styles from `pydata-sphinx-theme` and `sphinx-jupyterbook-theme`

### CODE REVIEW

File: quantecon_book_theme/layout.html [hunk 1]
```diff
@@ -176,35 +176,35 @@
...
-                <div class="sidebar__header">
+                <div class="qe-sidebar__header">
 
 
                     Contents
 
                 </div>
 
-                <nav class="sidebar__nav" id="sidebar-nav" aria-label="Main navigation">
+                <nav class="qe-sidebar__nav" id="qe-sidebar-nav" aria-label="Main navigation">
```
Comment (mmcky) [inline quantecon_book_theme/layout.html:191]: @AakashGfude is the `__` a style thing in `html`. Seems strange to me not to just use `-`

**Review by mmcky — APPROVED** [review]
@AakashGfude this looks like a really sensible thing to do to reduce the chance of conflicts with inherited styles.

File: quantecon_book_theme/layout.html (hunk 1 above)
Comment (AakashGfude) [reply quantecon_book_theme/layout.html:191]: true. I should change these as well I think.

Comment (AakashGfude) [reply quantecon_book_theme/layout.html:191]: @mmcky the `__` is used to represent the elements of a block, as mentioned in this article https://www.freecodecamp.org/news/css-naming-conventions-that-will-save-you-hours-of-debugging-35cea737d849/ . Makes sense to use this convention.

Comment (AakashGfude) [reply quantecon_book_theme/layout.html:191]: Maybe we should mention this BEM convention in contribution guidelines, for consistency.

**Comment by mmcky** [comment]
thanks @AakashGfude please go ahead when you're ready and I'll open an upgrade PR on lecture-python.myst

**Pull request merged**